### PR TITLE
Edit URL (API Added)

### PR DESCRIPTION
**Pull request by harshilsharmaa**
## Summary
API to Edit already existed URL 

Capture

### CODE REVIEW

File: server/src/controllers/links.controller.js [hunk 1]
```diff
@@ -33,12 +33,38 @@ createLink = async (req, res) => {
...
+    }
+}
+
+editLink = async (req, res) => {
+    try {
+        if(!req.body.url){
+            return res.status(500).json({ "success": false, "message": "URL is required." })
+        }
+
+        const link = await Link.findById(req.params.urlID)
```
Comment (cherylli) [inline server/src/controllers/links.controller.js:47]: we don't use the id. the short code would be enough as it's unique

**Review by cherylli — CHANGES_REQUESTED** [review]
is the code tested locally? Also, I would like to use the short code instead of the id. otherwise looks good. Thanks for updating the api docs as well

**Comment by harshilsharmaa** [comment]
API updated.
Now we have to just pass the short code in params.

API is tested and working fine.

**Comment by cherylli** [comment]
a suggestion for next time is not to use main to merge into main, it was a bit confusing when resolving conflicts...

**Pull request merged**